google/ggrc-core
# GGRC-6383 Create controls directly instead of using jQuery pluginName

# Issue description

CanJS extends jQuery with `pluginName` method which initiates CanJS control.

# Steps to test the changes

Affected places should work properly: **dashboard,  info pin, lhn, infinite scroll in lhn, all modals, administration tree-view, generic creation and edit flow.**

# Solution description

Replace usage of `jQuery.pluginName` with `new Control()`.

# Sanity checklist




# PR Review checklist
